packages/battery/lib/battery.dart

```diff
@@ -45,7 +45,8 @@ class Battery {
   }
 }
 
-BatteryState _stringToBatteryStateEnum(String state) {
+BatteryState _stringToBatteryStateEnum(dynamic event) {
```

Comment:
The method should probably be renamed now.

Reply:
I pulled the dynamic/string conversion into an anonymous function at the call site.